### PR TITLE
azure.md: config snippet for using PTP RTC

### PR DESCRIPTION
**Pull request by t-lo**
This change adds a config snippet to the Azure documentation for using a minimal chrony container instead of systemd-timesyncd. Chrony is set up to use Azure's virtual RTC.

Closes https://github.com/flatcar-linux/Flatcar/issues/686 .

### CODE REVIEW

File: docs/installing/cloud/azure.md [hunk 1]
```diff
@@ -221,6 +221,75 @@ Transpile it to Ignition JSON:
...
+      contents: |
+        [Unit]
+        Description=Build the chrony container image
+        ConditionPathExists=!/opt/chrony-build/done
+        [Service]
+        Type=oneshot
+        RemainAfterExit=true
+        Restart=on-failure
+        WorkingDirectory=/opt/chrony-build
+        ExecStart=docker build -t chrony .
```
Comment (pothos) [inline docs/installing/cloud/azure.md:265]: For some reason, sometimes, using a relative path fails and an absolute path was required - not sure if that was an issue of the past or may happen again

Comment (t-lo) [reply docs/installing/cloud/azure.md:265]: Good catch, will update.

Comment (t-lo) [reply docs/installing/cloud/azure.md]: Addressed, thanks for catching this.

File: docs/installing/cloud/azure.md [hunk 2]
```diff
@@ -221,6 +221,75 @@ Transpile it to Ignition JSON:
...
+        [Unit]
+        Description=Build the chrony container image
+        ConditionPathExists=!/opt/chrony-build/done
+        [Service]
+        Type=oneshot
+        RemainAfterExit=true
+        Restart=on-failure
+        WorkingDirectory=/opt/chrony-build
+        ExecStart=/usr/bin/docker build -t chrony .
+        ExecStartPost=touch done
```
Comment (pothos) [inline docs/installing/cloud/azure.md:266]:
```suggestion
        ExecStartPost=/usr/bin/touch done
```

**Review by pothos — APPROVED** [review]

**Pull request merged**